MediaBrowser.Providers/MediaInfo/FFProbeVideoInfo.cs

```diff
@@ -354,7 +354,7 @@ private void FetchBdInfo(Video video, ref ChapterInfo[] chapters, List<MediaStre
                 blurayVideoStream.Codec = ffmpegVideoStream.Codec;
                 blurayVideoStream.BitRate = blurayVideoStream.BitRate.GetValueOrDefault() == 0 ? ffmpegVideoStream.BitRate : blurayVideoStream.BitRate;
                 blurayVideoStream.Width = blurayVideoStream.Width.GetValueOrDefault() == 0 ? ffmpegVideoStream.Width : blurayVideoStream.Width;
-                blurayVideoStream.Height = blurayVideoStream.Height.GetValueOrDefault() == 0 ? ffmpegVideoStream.Width : blurayVideoStream.Height;
```

Comment:
This breaks current code and DB isn't it? Now it is designed to have wrong value stored

Reply:
Not sure what you're asking here. Yes in some rare cases the wrong video height could have been saved to the DB. Personally I haven't seen any such instances. With this change a user can simply rescan any media where the wrong resolution was saved to fix them.